apps/api/src/hyp3_api/validation.py

```diff
@@ -209,6 +216,51 @@ def check_bounding_box_size(job: dict, _, max_bounds_area: float = 4.5) -> None:
         )
 
 
+def _has_opera_rtc_static_coverage(granule_name: str) -> bool:
+    burst_number, swath = granule_name.split('_')[1:3]
+    params = {
+        'short_name': 'OPERA_L2_RTC-S1-STATIC_V1',
+        'granule_ur': f'OPERA_L2_RTC-S1-STATIC_T*-{burst_number}-{swath}_*',
+        'options[granule_ur][pattern]': 'true',
+    }
+    response = requests.get(CMR_URL, params=params)
+    response.raise_for_status()
+    return bool(response.json()['feed']['entry'])
+
+
+def check_opera_rtc_static_coverage(job: dict, _) -> None:
+    granules = job['job_parameters']['granules']
+    if len(granules) != 1:
+        raise InternalValidationError(f'Expected 1 granule, got {granules}')
+
+    granule = granules[0]
+    if not _has_opera_rtc_static_coverage(granule):
+        raise GranuleValidationError(f'Granule {granule} is outside of the OPERA RTC processing extent.')
+
+
+def check_opera_rtc_date(job: dict, _) -> None:
+    granules = job['job_parameters']['granules']
+    if len(granules) != 1:
+        raise InternalValidationError(f'Expected 1 granule, got {granules}')
+
+    granule = granules[0]
```

Comment:
`granules` as a hedge for the future feels strange to me.

Reply:
Yep, fair. @forrestfwilliams said there might be a 5% chance we would allow multiple granules with this job type in the future, so I think I'm reluctant to intentionally restrict ourselves to a single granule by changing the job spec parameter to a `granule` string. The experience with attempting to convert our single-burst InSAR job type to multi-burst InSAR and realizing that we needed to break backwards compatibility has made me slightly more in favor of future-proofing ourselves in these situations. But I'm on the fence and could be convinced.